src/routes/expenses.router.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const express = require('express');
+
+const expensesController = require('../controllers/expenses.controller');
+const expensesRouter = express.Router();
+
+expensesRouter.get('/', expensesController.getAllExpenses);
+expensesRouter.post('/', expensesController.createExpense);
+expensesRouter.get('/:id', expensesController.getOneExpense);
+expensesRouter.patch('/:id', expensesController.updateExpense);
+expensesRouter.delete('/:id', expensesController.removeExpense);
```

Comment:
You can use the middleware before the controller and sending the 400 error in the middleware if needed